apps/client/src/app/dashboard/page.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { Button, Card } from "@spin-spot/components";
+import { useRouter } from "next/navigation";
+
+export default function Dashboard() {
+  const router = useRouter();
+
+  const handleClick = () => {
+    router.push("/tables");
+  };
+  return (
+    <div className="flex-grow">
+      <div className="grid h-56 w-full items-center justify-items-center bg-black md:h-60">
+        <div className="font-quicksand text-center font-normal">
+          <h1 className="text-2xl">PING PONG</h1>
+          <h1 className="text-2xl">UNIMET</h1>
+        </div>
```

Comment:
Sólo debería haber 1 etiqueta <h1> en cada página. Puedes envolver todo el título en una sola y los organizas en un flex como dos <span> diferentes
```suggestion
        <h1 className="font-title text-center text-2xl flex flex-col">
          <span>PING PONG</span>
          <span>UNIMET</span>
        </h1>
```

Reply:
Listo esto por aquí también.